tst/EnergyPlus/unit/StandardRatings.unit.cc

```diff
@@ -1175,12 +1179,14 @@ TEST_F(EnergyPlusFixture, ChillerIPLVTestWaterCooled)
                     state->dataChillerElectricEIR->ElectricEIRChiller(1).ChillerEIRFTIndex,
                     state->dataChillerElectricEIR->ElectricEIRChiller(1).ChillerEIRFPLRIndex,
                     state->dataChillerElectricEIR->ElectricEIRChiller(1).MinUnloadRat,
-                    IPLV,
+                    IPLVSI,
+                    IPLVIP,
                     ObjexxFCL::Optional<const Real64>(),
                     ObjexxFCL::Optional_int_const(),
                     ObjexxFCL::Optional<const Real64>());
 
-    EXPECT_DOUBLE_EQ(round(IPLV * 100) / 100, 5.44); // 18.56 IPLV
+    EXPECT_DOUBLE_EQ(round(IPLVSI * 100) / 100, 5.44); // 5.44 IPLV.SI (AHRI Std 551/591 Test Condition)
+    EXPECT_DOUBLE_EQ(round(IPLVIP * 100) / 100, 5.47); // 18.66 IPLV.IP (AHRI Std 550/590 Test Condition)
 }
```

Comment:
Same here. The difference between `IPLV.IP` and `IPLV.SI` (when reported in the smae units) are due to changes in the AHRI Std test conditions only.